lib/imports.js

```diff
@@ -1,13 +1,65 @@
 import {
   createImportDeclaration,
   createNamedImportSpecifier,
-} from './createNode';
+} from "./createNode";
+
+const helpers = {
+  isExclusivelyDefaultImport: (importNode) =>
+    importNode.specifiers.length === 1 &&
+    importNode.specifiers[0].type === "ImportDefaultSpecifier",
```

Comment:
this needs a JSDoc. 

Also, could you keep the parenthesis around the whole thing as originally written, please?
I need to hook it in to the eslint finally.